examples/producer/main.go

```diff
@@ -92,7 +94,7 @@ RETRY:
 						Resource:  pub.Resource,
 						DataType:  cneevent.NOTIFICATION,
 						ValueType: cneevent.ENUMERATION,
-						Value:     cneevent.ACQUIRING_SYNC,
+						Value:     ptp.ACQUIRING_SYNC,
```

Comment:
here too event type is not set to right enum ,  it is set to event.Type = "ptp_status_type"

Reply:
done